UBC-MDS/explorePy
# Updated tests and fixed a few minor typos in the function.py files

I've updated the test files and the tests now seem to all pass with the outputs of our functions. Yay 👍 

**How to run Tests:**

1) Navigate to the tests folder on your local copy of the repository once you have pulled the latest files

2) Type pytest

_*If you see an error of the sort:_
AttributeError: 'Function' object has no attribute 'get_marker'

You might need to update the remotedata plugin for pytest. The following command in terminal fixed it for me:

`pip install --upgrade pytest-remotedata`
